src/distribution/Microsoft.Native.Quic.MsQuic.openssl.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Native.Quic.MsQuic.OpenSSL</id>
+    <version>0.0.0</version>
```

Comment:
@nibanks should this reflect the MsQuic version?

Reply:
intuitively it seems like it ought to, but the quictls nuspec file also leaves this at 0.0.0, so perhaps theres some command line override during packaging here?

Reply:
It's populated dynamically later. It's correct as is.